ember_debug/libs/view-inspection.js

```diff
@@ -1,8 +1,6 @@
+import { classify } from '@ember/string';
 import bound from 'ember-debug/utils/bound-method';
 
-const Ember = window.Ember;
```

Comment:
I thought we had to use `window.Ember` so we can use the Ember from the app?

Reply:
This was only used for destructuring `classify`